File: algorithms/implementation/drawing-book/drawing-book-javascript/js/test_pageCount.js
import {pageCount} from '../js/pageCount.js';
import assert from 'assert';

describe('pageCount', function() {

    it('O número mínimo de páginas para virar é 0', function() {

        let n = 5;
        let p = 4;

        let esperado = 0;

        let resultado = pageCount(n, p);

        assert.strictEqual(resultado, esperado)
    });
});

describe('pageCount', function() {

    it('O número mínimo de páginas para virar é 1', function() {

        let n = 6;
        let p = 2;

        let esperado = 1;

        let resultado = pageCount(n, p);

        assert.strictEqual(resultado, esperado)
    });
});

File: algorithms/implementation/drawing-book/drawing-book-python/tests/test_pageCount.py
from pageCount import pageCount

class TestePageCount:
    
    def testeCaso1(self):
        
        n = 5
        p = 4
        
        esperado = 0
        
        resultado = pageCount(n, p)
        
        assert resultado == esperado
        
    def testeCaso2(self):
        
        n = 6
        p = 2
        
        esperado = 1
        
        resultado = pageCount(n, p)
        
        assert resultado == esperado